utilities/tools/Set-ModuleReadMe.ps1

```diff
@@ -166,7 +166,7 @@ function Set-ParametersSection {
         # 2. Create header including optional columns
         $newSectionContent += @(
             ('**{0} parameters**' -f $category),
-            (''),
+            '',
```

Comment:
We should run this on all module readmes to double check if updates would be introduced and avoid static validations to fail. Was that done?

Reply:
Yes. By @ahmadabdalla in a PR that was merged yesterday. The above is just removing some redundant columns

Reply:
Added a fix that cause the 2 changed files to disappear